relay-server/src/services/outcome.rs

```diff
@@ -205,6 +205,9 @@ impl Outcome {
     /// Returns the `reason` code field of this outcome.
     pub fn to_reason(&self) -> Option<Cow<'_, str>> {
         match self {
+            Outcome::Invalid(DiscardReason::TooLarge(too_large_reason)) => Some(Cow::Owned(
+                format!("too_large: {}", too_large_reason.as_str()),
```

Comment:
Also we probably want to change this to:
```suggestion
                format!("too_large_{}", too_large_reason.as_str()),
```